development/nautobot_config.py

```diff
@@ -310,6 +310,7 @@ def is_truthy(arg):
     "nautobot_chatops_ipfabric": {
         "IPFABRIC_API_TOKEN": os.environ.get("IPFABRIC_API_TOKEN"),
         "IPFABRIC_HOST": os.environ.get("IPFABRIC_HOST"),
+        "IPFABRIC_VERIFY": os.environ.get("IPFABRIC_VERIFY", True),
```

Comment:
Could you wrap that with the `is_truthy` function for safety.
```suggestion
        "IPFABRIC_VERIFY": is_truthy(os.environ.get("IPFABRIC_VERIFY", True)),
```